src/button/button.style.tsx

```diff
@@ -7,7 +7,7 @@ import { MainStyleProps } from "./types";
 
 export const Main = styled.button<MainStyleProps>`
     padding: 0.5rem 1rem;
-    min-width: 4rem;
+    min-width: 5.375rem;
```

Comment:
min-width is changed here because on figma the width of the edit button is changed to 52px. I changed the min-width to:5.375 rem which translates to 16px padding left + 16px padding right + 52px width = 84 px on developer tools

Reply:
have clarified with UX that we can follow the existing min width